go/mysql/binlog_event_json.go

```diff
@@ -36,23 +54,49 @@ func jlog(tpl string, vals ...interface{}) {
 	if !jsonDebug {
 		return
 	}
-	fmt.Printf(tpl+"\n", vals...)
+	log.Infof("JSON:"+tpl+"\n", vals...)
 	_ = printASCIIBytes
 }
 
 func printASCIIBytes(data []byte) {
 	if !jsonDebug {
 		return
 	}
-	fmt.Printf("\n\n%v\n[", data)
+	s := ""
 	for _, c := range data {
 		if c < 127 && c > 32 {
-			fmt.Printf("%c ", c)
+			s += fmt.Sprintf("%c ", c)
 		} else {
-			fmt.Printf("%02d ", c)
+			s += fmt.Sprintf("%02d ", c)
```

Comment:
This concatenation will end up inefficient. Consider using `strings.Builder`

Reply:
Actually this is only used for local debugging of the functionality. It is off in production. It will be removed soon once we have a few more prod tests with the json data type.

Reply:
ah, cool. In that case, keep it as it is.